pkg/storageincentives/agent.go

```diff
@@ -472,3 +487,15 @@ func (a *Agent) wrapCommit(storageRadius uint8, sample []byte, key []byte) ([]by
 func (a *Agent) Status() (*Status, error) {
 	return a.state.Status()
 }
+
+func (a *Agent) canCommit(ctx context.Context) (bool, error) {
+	balance, err := a.backend.BalanceAt(ctx, a.state.ethAddress, nil)
+	if err != nil {
+		return false, err
+	}
+
+	minBalance := big.NewInt(minTxCountToCover)
+	minBalance.Mul(minBalance, a.state.AvgFee())
```

Comment:
the avgFee is zero at the beginning, maybe there should be some minimum default value instead of zero.

Reply:
also the avgFee may lose it's accuracy because these transactions only occur every few days, ideally what we want is to fetch the current gas price from the backend and multiply with some avg gas limit (this number is stable between some range)

Reply:
see gas limits https://gnosisscan.io/address/0x8c26b7ca61a6608b011cba43d8ca4476b6d8da17